portfolio/src/main/webapp/index.html

```diff
@@ -5,73 +5,68 @@
+            <p class="sub-title"> Background </p>
+            <img class="mountain-tile-logo" src="images/mountain.png" alt="mountain">
           </div>
-          <div class="tileBack">
-            I was born in Sacramento, CA and have spent most of my life living there. Ever since I was little I have
-            been interested
-            in building and experimenting with different things, working on crafts with my mom and spending lots of time
-            figuring out how I can fit projects together.
-            In high school I joined an FRC team (robotics team) where I discovered my passion for programming and lead
-            the team to many successful competitions. I am currently
-            a rising junior at UCLA where I am involved in the UAS team (drone club) and also pursue research in NLP
-            through the Intel AI Student Ambassadors program
-
-            For fun I enjoy spending as much time as possible with my lovely dog, Bailee (I am a huge animal lover) as
-            well as attending as many sporting events as I can. I love going
-            with a group of friends to all UCLA's basketball and football as we root them on even when our teams are not
-            the best. When I'm pursuing these activities I like to spend time
-            working on OS and hardware projects, building a server for my home and trying to buy old computers to use in
-            new systems around my house.
+          <div class="tile-back">
+            <link rel="import" type="text/html" href="content/background.html" />
 
           </div>
         </div>
       </div>
       <div class="tile">
-        <div class="tileInner">
-          <div class="tileFront">
-            <p class="SubTitle"> More info </p>
+        <div class="tile-inner">
+          <div class="tile-front">
+            <p class="sub-title"> Personal Projects </p>
+            <img id="hammer-tile-logo" src="images/hammerremoved.png" alt="hammer">
           </div>
-          <div class="tileBack">
-            <p class="SubTitle"> Here is the info </p>
+          <div class="tile-back">
+            <p class="sub-title"> Projects Include </p>
+            <ul>
+              <li> Created Home Samba Server </li>
+              <li> Household PXE Network Boot </li>
+              <li> NLP Research through Intel AI</li>
+              <li>UAS Controls Obstacle Avoidance</li>
+              <li> Group Router Project</li>
+            </ul>
           </div>
         </div>
       </div>
-    <!--Would really be ideal to implement this tiles in React as they are getting somewhat redundant to construct-->
-    <div class="tile">
-      <div class="tileInner">
-        <div id="rFrontTile" class="tileFront">
-          <p class="SubTitle"> Fun Photos </p>
-        </div>
-        <div id="rBackTile" class="tileBack">
-          <img src="images/random/test1.jpg">
+      <!--TODO: implement these tiles in React as they are becoming redundant to construct-->
```

Comment:
You can add your username or issue id in TODO.

Reply:
Ok, I will do that to all of them